environments/64-test/public.yml

```diff
@@ -0,0 +1,177 @@
+# For a real deployment
+# should be e.g. 'www.commcarehq.org'
+internal_domain_name: 'localdomain'
+
+etc_hosts_lines: []
+etc_hosts_lines_removed: []
+
+touchforms_enabled: false
+
+riak_backend: "bitcask"
+riak_ring_size: 64
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
```

Comment:
not related to this PR but I think we could dynamically generate this now: https://stackoverflow.com/questions/19292899/creating-a-new-user-and-password-with-ansible#29910294

Reply:
Nice! Yeah we should use that